collect_app/src/androidTest/java/org/odk/collect/android/support/pages/Page.kt

```diff
@@ -557,6 +557,17 @@ abstract class Page<T : Page<T>> {
         return destination
     }
 
+    fun selectTextField(label: String, index: Int = 0): T {
+        closeSoftKeyboard()
```

Comment:
If you close the keyboard before clicking on another text field then this test does not cover the original scenario anymore and doesn't make sense.
The original issue was that:
1. There was a long list of questions.
2. The first question was a text question.
3. The last question was a numeric question (so that the keyboard type is different and will trigger calling `onLayoutChange` on `odkview` once you switch from the first one to the last one).
4. You add an answer to the first one (so that the text keyboard is open)
5. Then you click on the last question (and the keyboard needs to be updated which as I said above calls `onLayoutChange` on `odkview`).

If you close the keyboard between steps 4 and 5 then `onLayoutChange` (what was the cause of the issue) will never be called.

Reply:
Ah right! Well this is not a good solution then 😆. Back to the drawing board.